### DISCUSSION TITLE
where is train.idx  train.rec?

### DISCUSSION THREAD

**Discussion by jianyangshi** [leondgarse/Keras_insightface#87] [General]
""" CUDA_VISIBLE_DEVICES='-1' ./prepare_data.py -D /datasets/faces_emore """
""" CUDA_VISIBLE_DEVICES='-1' ./prepare_data.py -D /datasets/faces_emore -T lfw.bin cfp_fp.bin agedb_30.bin """

i can not get train.idx ,  train.rec, lfw.bin, cfp_fp.bin and agedb_30.bin
can you tell me how to get these?

**Comment by leondgarse** [comment]
Training Data in this project is downloaded from [Insightface Dataset Zoo](https://github.com/deepinsight/insightface/tree/master/recognition/_datasets_). It's the first line of `Beforehand Data Prepare`.

**Reply by jianyangshi** [reply]
Fine ! Thanks!
